### PR TITLE
Smoke test

### PR DESCRIPTION
**Pull request by AnidaLB**
RSpec syntax change

### CODE REVIEW

File: assert_spec.rb [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  config.color = true
+  config.formatter = :documentation
+  config.before(:each) { @browser = browser }
+  config.before(:all) {browser.window.maximize}
+  config.after(:suite) { browser.close unless browser.nil? }
+end
+
+
+
+describe 'Navigator test' do
```
Comment (harisatlant) [inline assert_spec.rb]: Change to 'Navigator smoke test'

File: assert_spec.rb [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+	context "When searching for a place" do
+
+
+it 'should open the main page' do
+ @browser.goto('http://www.navigator.ba/#/categories')
```
Comment (harisatlant) [inline assert_spec.rb]: This step should be divided into two steps. In first one, script will open 'navigator.ba' and second one will be assertion checking if page is correctly redirected to 'http://www.navigator.ba/#/categories'

File: assert_spec.rb [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+end 
+
+it 'should enter the name of the place' do
+	@browser.text_field(:xpath => '//*[@id="ember558"]').send_keys "atlantbh"
+end
+
```
Comment (harisatlant) [inline assert_spec.rb]: I suggest you add assertion here checking if object is present in the list.

File: assert_spec.rb [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+it 'should click on search icon' do
+	@browser.span(:xpath => '//*[@id="ember549"]/a/span').click
+end
+
+it 'should open the place from search results' do
+	@browser.element(title: "Atlantbh").click
+
+sleep 3
```
[the remark this answers is not in the document]
Comment (AnidaLB) [reply assert_spec.rb]: What if it's present immediately and I want a break of 3 seconds?

File: assert_spec.rb (hunk 2 above)
Comment (AnidaLB) [reply assert_spec.rb]: Also done

File: assert_spec.rb (hunk 4 above)
[the remark this answers is not in the document]
Comment (harisatlant) [reply assert_spec.rb]: Why do you need a break of 3 seconds?

[the remark this answers is not in the document]
Comment (AnidaLB) [reply assert_spec.rb]: For presentation purposes

**Pull request merged**